src/reports/filters/AppointmentFilter.tsx

```diff
@@ -0,0 +1,90 @@
+import {StyledForm} from "blaise-design-system-react-components";
+import React, {ReactElement} from "react";
+import dateFormatter from "dayjs";
+import utc from "dayjs/plugin/utc";
+import timezone from "dayjs/plugin/timezone";
+import Breadcrumbs from "../../components/Breadcrumbs";
+import AppointmentResourceDaybatchWarning from "../AppointmentResourcePlanning/AppointmentResourceDaybatchWarning";
+
+dateFormatter.extend(utc);
```

Comment:
These look like something that will affect everything that's been run subsequently. Perhaps this should be move to the initialisation of the app (e.g. into `index.ts` or similar)?